src/FluentValidation.Tests.AspNetCore/ServiceCollectionExtensionTests.cs

```diff
@@ -0,0 +1,30 @@
+﻿#if NETCOREAPP || NETSTANDARD

```

Comment:
I initially placed this in the Tests project so I had to restrict it to core/standard. I forgot to remove it after I moved to to this project.